### ISSUE TITLE
Talk: Make your applications faster by using AWS JavaScript SDK v3

### ISSUE THREAD

**Issue by trivikr** [PDXNode/pdxnode#107]
## Description:
The AWS SDK for JavaScript enables developers to build libraries an applications that use AWS services. You can use the JavaScript API in the browser and inside Node.js applications on the server. The v3 is now modular and has been written in TypeScript. In this talk, we'll understand how to make your application faster by using this modularized version of AWS JavaScript SDK

I'm a Software Development Engineer with AWS, and maintain [AWS SDK for JavaScript](https://docs.aws.amazon.com/sdk-for-javascript/index.html)

* Trivikram Kamat
* [Twitter](https://twitter.com/trivikram)
* [Github](https://github.com/trivikr)
* [Website](http://trivikr.github.io)

**Comment by MBerka** [comment]
Hi @trivikr, thank you for submitting this! We are always seeking new topics.
1. Does the talk have a beginner-friendly Node.js example? Some front-end is alright, but we try to emphasize the parts that other groups dealing with traditional JS might omit.
2. Would you deliver this in person, and in what timeframe? Presentation nights are usually the second Thursday each month and I do not know how much travel would be involved for you. Remote presentations are possible but may take more preparation.

**Comment by trivikr** [comment]
> Does the talk have a beginner-friendly Node.js example?

Out talk will have both Node.js and JavaScript examples for the backend and frontend respectively. Some basic knowledge of AWS services would be helpful, but we'll also cover what services are being used. For example:
* Slide with a summary of AWS SDK for JavaScript ([#6](https://slides.com/trivikram/modularized-aws-js-sdk#/6))
* Recording of the example application ([#8](https://slides.com/trivikram/modularized-aws-js-sdk#/8))
* Brief description of what DynamoDB is ([#9](https://slides.com/trivikram/modularized-aws-js-sdk#/9))
* Code sample for one of the lambdas in the backend ([#13](https://slides.com/trivikram/modularized-aws-js-sdk#/13))
* Code sample for one of the S3 function calls in the frontend ([#36](https://slides.com/trivikram/modularized-aws-js-sdk#/36))

I already gave this talk in BellevueJS September meetup ([event link](https://www.meetup.com/BellevueJS/events/llvkklyzmbgc/)) and my colleagues will be giving this talk at SeattleJS on Oct 10th ([event link](https://www.meetup.com/seattlejs/events/clmbzqyznbnb/))

> Would you deliver this in person?

I would like my colleagues @AllanFly120 and @ajredniwja to give this talk who're presenting at SeattleJS on Oct 10th ([details](https://github.com/seattlejs/seattlejs/issues/176)). We'll confirm the presenters only after this talk is approved by PDXNode.

> Timeframe for the talk? About travel involved and remote presentations.

We're looking for Thursday Jan 9th ([link](https://www.meetup.com/pdxnode/events/mldsxqybccbmb/)). The idea is to drive to Portland from Seattle on the day of the event and drive back in the night, but we'll definitely prefer remote presentation if the option is available.

**Comment by trivikr** [comment]
@MBerka What's the estimated number of attendees in PDXNode meetup?

**Comment by MBerka** [comment]
@trivikr 30-40 at recent presentation nights.

**Comment by MBerka** [comment]
Sorry, catching up after business travel. We would be happy to have you Jan 9th! Still making sure about the AV equipment; we'll spare you the drive if possible.

**Comment by trivikr** [comment]
Thanks @MBerka for the response.
For an audience of 30-40, we'll prefer to give talk remotely.

**Comment by trivikr** [comment]
Hi @MBerka, can we move this to February?

**Comment by MBerka** [comment]
Alright, it should work out with our speaker line-up. I have verified that the projector is accessible. Do your colleagues have preferred presentation software that we could try beforehand?

**Comment by trivikr** [comment]
> Alright, it should work out with our speaker line-up.

Thanks! I noticed that February meetup is on 02/13 https://www.meetup.com/pdxnode/events/mldsxqybcdbrb/
And the speaker for PDXNode talk @srchase is planning to speak at VanJS in Vancouver BC on that day https://github.com/VanJS/meetup/issues/30

@srchase Would you like to give this talk in March instead?
Or will @AllanFly120 be willing to give this talk in February?

> Do your colleagues have preferred presentation software that we could try beforehand?

For slides, we plan to use updated version of deck https://slides.com/trivikram/modularized-aws-js-sdk

For video chat software, I'll let @srchase or @AllanFly120 respond with their preference.

**Comment by trivikr** [comment]
Hi @MBerka, I'm closing this issue as of now - as we don't be able to do this in February.

I'll create a new issue in future closer to the meetup date.

**Comment by MBerka** [comment]
Hi @trivikr, thanks for letting us know. Someone else may be handling the matter in the future. Good luck in the meantime, especially at the conferences!
